user_guide/gherkin_language.rst

```diff
@@ -0,0 +1,105 @@
+About Gherkin Language
```

Comment:
I would rather name the chapter ``The Gherking Language`` rather than using ``About`` (and even if you keep ``About``, the title should still be ``About the Gherkin Language`` IMO)